### PR TITLE
validate_required_fields, incl. (adapted) tests

### PR DESCRIPTION
**Pull request by r-peschke**
Checks the required attribute of the models.py.
Preliminary: [PR adjusting required attributes for mediafile.upload](https://github.com/OpenSlides/OpenSlides/pull/5934)
This PR is a precondition for [PR meeting.create](https://github.com/OpenSlides/openslides-backend/pull/486)

### CODE REVIEW

File: tests/system/action/test_relation_cud_required_variations.py [hunk 1]
```diff
@@ -0,0 +1,243 @@
...
+    schema = {}  # type: ignore
+
+
+@register_action("fake_model_b.update")
+class FakeModelAUpdateAction(UpdateAction):
+    model = FakeModelB()
+    schema = {}  # type: ignore
+
+
+class TestDeleteVariations(BaseActionTestCase):
```
Comment (jsangmeister) [inline tests/system/action/test_relation_cud_required_variations.py:104]: These should already be tested in `test_cascade_deletion.py` - at least the first two are doubled.

File: tests/system/action/test_relation_cud_required_variations.py [hunk 2]
```diff
@@ -0,0 +1,243 @@
...
+        self.assert_status_code(response, 200)
+        self.assert_model_exists("fake_model_a/1", {"fake_b_setnul_to_req_id": None})
+        self.assert_model_exists("fake_model_a/2", {"fake_b_setnul_to_req_id": 1})
+        self.assert_model_exists("fake_model_b/1", {"fake_a_setnul_to_req_ids": [2]})
+
+
+class TestCreateVariations(BaseActionTestCase):
+    def test_create_a_impossible(self) -> None:
+        """
+        Should be impossible, because fake_b_casc_to_all_id can't be set, because there is no fake_model_b instance stored.
```
Comment (jsangmeister) [inline tests/system/action/test_relation_cud_required_variations.py:214]: Yes, this is true, but in practice this situation will never occur, since we always have the base model `organisation/1`. All relations are rooted from there and this model is created outside of the backend-action-context. So in practice, all required relations will always be able to be set.

**Comment by jsangmeister** [comment]
In https://github.com/OpenSlides/openslides-backend-coordination/issues/109 it is listed what models have to be create when a meeting is created.

File: tests/system/action/test_relation_cud_required_variations.py (hunk 1 above)
Comment (r-peschke) [reply tests/system/action/test_relation_cud_required_variations.py:104]: Then let us group them together by the criterias create/update/delete,  cascade/protect/set_null and required/not required, whereas required only has an impact on set_null.

Comment (jsangmeister) [reply tests/system/action/test_relation_cud_required_variations.py:104]: Sure, you can go ahead and do that.

**Comment by r-peschke** [comment]
Failing tests with activated required-check:

======================== short test summary info all required checks active =========================================================
FAILED tests/system/action/assignment/test_create.py::AssignmentCreateActionTest::test_create - AssertionError: 400 != 200
FAILED tests/system/action/assignment/test_create.py::AssignmentCreateActionTest::test_create_agenda_item_no_default - AssertionError: 400 != 200
FAILED tests/system/action/assignment/test_create.py::AssignmentCreateActionTest::test_create_full_fields - AssertionError: 400 != 200
FAILED tests/system/action/assignment/test_create.py::AssignmentCreateActionTest::test_create_other_agenda_item_check - AssertionError: 400 != 200
FAILED tests/system/action/assignment/test_create.py::AssignmentCreateActionTest::test_create_other_agenda_item_check_2 - AssertionError: 400 != 200
FAILED tests/system/action/assignment/test_create.py::AssignmentCreateActionTest::test_create_other_agenda_item_check_3 - AssertionError: 400 != 200
FAILED tests/system/action/mediafile/test_create_directory.py::MediafileCreateDirectoryActionTest::test_create_directory_correct - AssertionError: 400 != 200
FAILED tests/system/action/mediafile/test_create_directory.py::MediafileCreateDirectoryActionTest::test_create_directory_parent - AssertionError: 400 != 200
FAILED tests/system/action/mediafile/test_create_directory.py::MediafileCreateDirectoryActionTest::test_create_directory_parent_case1 - AssertionError: 400 != 200
FAILED tests/system/action/mediafile/test_create_directory.py::MediafileCreateDirectoryActionTest::test_create_directory_parent_case2 - AssertionError: 400 != 200
FAILED tests/system/action/mediafile/test_create_directory.py::MediafileCreateDirectoryActionTest::test_create_directory_parent_case3 - AssertionError: 400 != 200
FAILED tests/system/action/mediafile/test_create_directory.py::MediafileCreateDirectoryActionTest::test_create_directory_parent_case4 - AssertionError: 400 != 200
FAILED tests/system/action/mediafile/test_create_directory.py::MediafileCreateDirectoryActionTest::test_create_directory_parent_case5 - AssertionError: 400 != 200
FAILED tests/system/action/mediafile/test_create_directory.py::MediafileCreateDirectoryActionTest::test_create_directory_parent_inherited_list - AssertionError: 400 != 200
FAILED tests/system/action/mediafile/test_upload.py::MediafileUploadActionTest::test_create - AssertionError: 400 != 200
FAILED tests/system/action/mediafile/test_upload.py::MediafileUploadActionTest::test_create_access_group - AssertionError: 400 != 200
FAILED tests/system/action/mediafile/test_upload.py::MediafileUploadActionTest::test_upload_pdf - AssertionError: 400 != 200
FAILED tests/system/action/meeting/test_create.py::MeetingCreateActionTest::test_check_payload_fields - AssertionError: 400 != 200
FAILED tests/system/action/meeting/test_create.py::MeetingCreateActionTest::test_create_simple - AssertionError: 400 != 200
FAILED tests/system/action/motion/test_create.py::MotionCreateActionTest::test_correct_origin_id_set - AssertionError: 400 != 200
FAILED tests/system/action/motion/test_create.py::MotionCreateActionTest::test_create_good_case_required_fields - AssertionError: 400 != 200
FAILED tests/system/action/motion/test_create.py::MotionCreateActionTest::test_create_simple_fields - AssertionError: 400 != 200
FAILED tests/system/action/motion/test_create.py::MotionCreateActionTest::test_create_with_submitters - AssertionError: 400 != 200
FAILED tests/system/action/motion/test_create.py::MotionCreateActionTest::test_create_workflow_id - AssertionError: 400 != 200
FAILED tests/system/action/motion/test_create.py::MotionCreateActionTest::test_create_workflow_id_from_meeting - AssertionError: 400 != 200
FAILED tests/system/action/motion/test_create_amendment.py::MotionCreateAmendmentActionTest::test_create_amendment - AssertionError: 400 != 200
FAILED tests/system/action/motion/test_create_amendment.py::MotionCreateAmendmentActionTest::test_create_amendment_default_workflow - AssertionError: 400 != 200
FAILED tests/system/action/motion/test_create_amendment.py::MotionCreateAmendmentActionTest::test_create_with_amendment_paragraphs_0 - AssertionError: 400 != 200
FAILED tests/system/action/motion/test_create_amendment.py::MotionCreateAmendmentActionTest::test_create_with_amendment_paragraphs_string - AssertionError: 400 != 200
FAILED tests/system/action/motion/test_create_amendment.py::MotionCreateAmendmentActionTest::test_create_with_amendment_paragraphs_valid - AssertionError: 400 != 200
FAILED tests/system/action/motion/test_create_sequential_number.py::MotionCreateActionTestSequentialNumber::test_create_sequential_numbers - AssertionError: 400 != 200
FAILED tests/system/action/motion/test_create_sequential_number.py::MotionCreateActionTestSequentialNumber::test_create_sequential_numbers_2meetings - AssertionError: 4...
FAILED tests/system/action/motion/test_create_sequential_number.py::MotionCreateActionTestSequentialNumber::test_create_sequential_numbers_deleted_motion - AssertionErr...
FAILED tests/system/action/motion/test_create_sequential_number.py::MotionCreateActionTestSequentialNumber::test_create_sequential_numbers_race_condition - openslides_b...
FAILED tests/system/action/motion/test_create_statute_amendment.py::MotionCreateAmendmentActionTest::test_create_statute_amendment - AssertionError: 400 != 200
FAILED tests/system/action/motion/test_create_statute_amendment.py::MotionCreateAmendmentActionTest::test_create_statute_amendment_default_workflow - AssertionError: 40...
FAILED tests/system/action/motion/test_set_number_mixin.py::MotionSetNumberMixinTest::test_create_set_number_get_number_per_category - AssertionError: 400 != 200
FAILED tests/system/action/motion/test_set_number_mixin.py::MotionSetNumberMixinTest::test_create_set_number_good - AssertionError: 400 != 200
FAILED tests/system/action/motion/test_set_number_mixin.py::MotionSetNumberMixinTest::test_create_set_number_min_digits - AssertionError: 400 != 200
FAILED tests/system/action/motion/test_set_number_mixin.py::MotionSetNumberMixinTest::test_create_set_number_prefix_blank_lead_motion - AssertionError: 400 != 200
FAILED tests/system/action/motion/test_set_number_mixin.py::MotionSetNumberMixinTest::test_create_set_number_prefix_blank_lead_motion_number_inc - AssertionError: 400 !...
FAILED tests/system/action/motion/test_set_number_mixin.py::MotionSetNumberMixinTest::test_create_set_number_return_because_number_preset - AssertionError: 400 != 200
FAILED tests/system/action/motion/test_set_number_mixin.py::MotionSetNumberMixinTest::test_create_set_number_return_because_number_type_manually - AssertionError: 400 !...
FAILED tests/system/action/motion/test_set_number_mixin.py::MotionSetNumberMixinTest::test_create_set_number_unique_check_jump - AssertionError: 400 != 200
FAILED tests/system/action/motion/test_set_number_mixin.py::MotionSetNumberMixinTest::test_set_number_false - AssertionError: 400 != 200
FAILED tests/system/action/motion/test_set_number_mixin.py::SetNumberMixinManuallyTest::test_complex_example_manually_1 - AssertionError: 400 != 200
FAILED tests/system/action/motion/test_set_number_mixin.py::SetNumberMixinManuallyTest::test_complex_example_manually_2 - AssertionError: 400 != 200
FAILED tests/system/action/motion/test_set_number_mixin.py::SetNumberMixinManuallyTest::test_complex_example_manually_3 - AssertionError: 400 != 200
FAILED tests/system/action/motion/test_set_number_mixin.py::SetNumberMixinSeriallTest::test_complex_example_serially_numbered_1 - AssertionError: 400 != 200
FAILED tests/system/action/motion/test_set_number_mixin.py::SetNumberMixinSeriallTest::test_complex_example_serially_numbered_2 - AssertionError: 400 != 200
FAILED tests/system/action/motion/test_set_number_mixin.py::SetNumberMixinSeriallTest::test_complex_example_serially_numbered_3 - AssertionError: 400 != 200
FAILED tests/system/action/motion/test_set_number_mixin.py::SetNumberMixinComplexExamplesPerCategoryTest::test_complex_example_per_category_1_1 - AssertionError: 400 !=...
FAILED tests/system/action/motion/test_set_number_mixin.py::SetNumberMixinComplexExamplesPerCategoryTest::test_complex_example_per_category_1_2 - AssertionError: 400 !=...
FAILED tests/system/action/motion/test_set_number_mixin.py::SetNumberMixinComplexExamplesPerCategoryTest::test_complex_example_per_category_2_1 - AssertionError: 400 !=...
FAILED tests/system/action/motion/test_set_number_mixin.py::SetNumberMixinComplexExamplesPerCategoryTest::test_complex_example_per_category_2_2 - AssertionError: 400 !=...
FAILED tests/system/action/motion/test_set_number_mixin.py::SetNumberMixinComplexExamplesPerCategoryTest::test_complex_example_per_category_2_3 - AssertionError: 400 !=...
FAILED tests/system/action/motion/test_set_number_mixin.py::SetNumberMixinComplexExamplesPerCategoryTest::test_complex_example_per_category_2_4 - AssertionError: 400 !=...
FAILED tests/system/action/motion_block/test_create.py::MotionBlockActionTest::test_create - AssertionError: 400 != 200
FAILED tests/system/action/motion_workflow/test_create.py::MotionWorkflowSystemTest::test_create - AssertionError: 400 != 200
FAILED tests/system/action/poll/test_create.py::PollCreateTest::test_create_correct - AssertionError: 400 != 200
FAILED tests/system/action/poll/test_create.py::PollCreateTest::test_create_three_options - AssertionError: 400 != 200
FAILED tests/system/action/topic/test_create.py::TopicSystemTest::test_create - AssertionError: 400 != 200
FAILED tests/system/action/topic/test_create.py::TopicSystemTest::test_create_more_fields - AssertionError: 400 != 200
FAILED tests/system/action/topic/test_create.py::TopicSystemTest::test_create_multi - AssertionError: "Datastore service sends HTTP 400. Model 'meeting/1' raises MODEL_...
FAILED tests/system/action/topic/test_create.py::TopicSystemTest::test_create_multiple - AssertionError: 400 != 200
FAILED tests/system/action/topic/test_delete.py::TopicDeleteActionTest::test_create_delete - AssertionError: 400 != 200
=================================== 66 failed, 596 passed in 167.63s (0:02:47) ========================

File: openslides_backend/action/action.py [hunk 3]
```diff
@@ -181,16 +185,39 @@ def handle_relation_updates(
         )
         fields: Optional[Dict[str, Any]]
         for fqfield, data in relation_updates.items():
+            field = getattr(model_registry[fqfield.collection], fqfield.field, None)
```
Comment (jsangmeister) [inline openslides_backend/action/action.py:188]: You can use `self.model.get_field` to access the field.

File: openslides_backend/action/action.py [hunk 4]
```diff
@@ -181,16 +185,39 @@ def handle_relation_updates(
...
+                    field
+                    and field.required
+                    and not data.get("value")
+                    and not isinstance(
+                        self.additional_relation_models.get(fqfield.fqid), DeletedModel
+                    )
+                ):
+                    raise RequiredFieldsException(
+                        f"From relation {fqfield.fqid}", [fqfield.field]
+                    )
```
Comment (jsangmeister) [inline openslides_backend/action/action.py:202]: I think by performing the check here you are missing cases where an action directly deletes a field which is required. Or is this caught somewhere else?

File: openslides_backend/action/generics/create.py [hunk 5]
```diff
@@ -50,3 +52,47 @@ def create_write_requests(
...
+        """
+        payload = {field: instance["id"] for field in fields}
+        payload.update(additional_dict)
+        self.execute_other_action(action_class, [payload])
+        fqid = FullQualifiedId(self.model.collection, instance["id"])
+        for wr in self.write_requests:
+            for event in wr.events:  # type: ignore
+                if event["fqid"] == fqid and event["type"] == EventType.Update:
+                    instance.update(event["fields"])  # type: ignore
+        return instance
```
Comment (jsangmeister) [inline openslides_backend/action/generics/create.py:98]: Mmh, I'm not a fan of this - it complicates the process of delegating actions additionally. We talked about checking the resulting write request events, did you try to take that approach?

**Review by jsangmeister — CHANGES_REQUESTED** [review]

**Comment by jsangmeister** [comment]
I took a quick look at the failing tests and most of them seem to be create action which call other create actions, which is currently not caught by the checking routine (e.g. `topic.create` has to create an agenda item and a list of speakers, but the tests fail because the field `agenda_item_id` and `list_of_speaker_id` are temporarily not set).

IMO this can be solved and the check furthermore simplified by not checking in each action call itself, but before writing the result to the datastore. You can iterate the events of the write request and check this way if any required fields would be left empty at the end of an action.

File: openslides_backend/action/action.py (hunk 3 above)
Comment (r-peschke) [reply openslides_backend/action/action.py:188]: It's not necessary the self.model, sometimes the collection from the other side of the relation

File: openslides_backend/action/generics/create.py (hunk 5 above)
Comment (r-peschke) [reply openslides_backend/action/generics/create.py]: replaced by the existing solution CreateActionWithDependencies

Comment (r-peschke) [reply openslides_backend/action/generics/create.py]: The validation took place by checking the resulting write request events

File: openslides_backend/action/action.py (hunk 3 above)
Comment (r-peschke) [reply openslides_backend/action/action.py]: doesn't exist anymore

**Review by FinnStutzenstein — APPROVED** [review]
I merged https://github.com/OpenSlides/OpenSlides/pull/5934. The PR is ok for me. I didn't looked into the code too much.

File: openslides_backend/action/action.py [hunk 6]
```diff
@@ -279,6 +283,52 @@ def process_write_requests(
...
+            instance = v["fields"]
+            required_fields = []
+            if type_ == EventType.Create:
+                required_fields = [
+                    field.own_field_name
+                    for field in model_registry[fqid.collection]().get_required_fields()
+                    if field.own_field_name not in instance
+                    or (
+                        field.own_field_name in instance
+                        and not instance[field.own_field_name]
```
Comment (jsangmeister) [inline openslides_backend/action/action.py:317]: This will also catch `""` and `0` which is probably not what we want.

File: openslides_backend/action/action_handler.py [hunk 7]
```diff
@@ -184,6 +184,8 @@ def perform_action(
 
         try:
             write_request, results = action.perform(action_data, self.user_id)
+            if isinstance(write_request, WriteRequest):
```
Comment (jsangmeister) [inline openslides_backend/action/action_handler.py:187]: This check seems unnecessary, it is always a WriteRequest; `if write_request` should suffice

Comment (jsangmeister) [reply openslides_backend/action/action_handler.py:187]: On another note, we could move this on level up if we want: Currently, if we have an action which executes another action, the required fields are checked one time in the inner action and then another time in the outer action. If we move the check to the `action_handler.py`, we save ourselves the redundancy.

File: openslides_backend/action/actions/group/create.py [hunk 8]
```diff
@@ -13,5 +13,9 @@ class GroupCreate(CreateAction):
     model = Group()
     schema = DefaultSchema(Group()).get_create_schema(
         required_properties=["name", "meeting_id"],
-        optional_properties=["permissions"],
+        optional_properties=[
+            "permissions",
+            "default_group_for_meeting_id",
+            "admin_group_for_meeting_id",
```
Comment (jsangmeister) [inline openslides_backend/action/actions/group/create.py:19]: Please remove this: this is not part of the action payload, as described in https://github.com/OpenSlides/OpenSlides/wiki/group.create. If you need this for internal reasons, you would have to create a separate action for this or (probably more complex) implement the possibility to mark some payload elements as internal. Also, on another note, this seems like it belongs to #486 and not this PR.

Comment (jsangmeister) [reply openslides_backend/action/actions/group/create.py:19]: Except, @FinnStutzenstein: Does the user have the option to change these default groups or are they fix once the meeting is created? If the former, we could talk about adding this to the payload.

File: openslides_backend/action/actions/motion_state/create_update_delete.py [hunk 9]
```diff
@@ -62,6 +62,7 @@ class MotionStateActionSet(ActionSet):
             "merge_amendment_into_final",
             "show_recommendation_extension_field",
             "first_state_of_workflow_id",
+            "next_state_ids",
```
Comment (jsangmeister) [inline openslides_backend/action/actions/motion_state/create_update_delete.py:65]: This seems fine for me, please document this in the wiki (https://github.com/OpenSlides/OpenSlides/wiki/motion_state.create) and add a test for it.

File: openslides_backend/action/mixins/create_action_with_dependencies.py [hunk 10]
```diff
@@ -20,7 +20,7 @@ def base_update_instance(self, instance: Dict[str, Any]) -> Dict[str, Any]:
         additional_relation_models = {
             FullQualifiedId(self.model.collection, instance["id"]): instance,
         }
-        for ActionClass in self.dependencies:
+        for index, ActionClass in enumerate(self.dependencies):
```
Comment (jsangmeister) [inline openslides_backend/action/mixins/create_action_with_dependencies.py:23]: This also seems like it doesn't belong in this PR. Also I think the cleaner solution would be the change the return type of the `payload_method` to an array of objects. Dealing with explicit indices is always a hassle...

File: openslides_backend/action/action_handler.py (hunk 7 above)
Comment (jsangmeister) [reply openslides_backend/action/action_handler.py:187]: The more I think about it the more I like it. Please tell me if you have any arguments against this.

**Review by jsangmeister — CHANGES_REQUESTED** [review]

File: openslides_backend/action/actions/group/create.py (hunk 8 above)
Comment (FinnStutzenstein) [reply openslides_backend/action/actions/group/create.py:19]: They are fixed and only touched on meeting.create

File: openslides_backend/action/action.py (hunk 6 above)
Comment (jsangmeister) [reply openslides_backend/action/action.py:317]: ok, we'll leave it as is for now as discussed.

File: tests/system/action/test_relation_cud_required_variations.py (hunk 1 above)
Comment (r-peschke) [reply tests/system/action/test_relation_cud_required_variations.py:176]: Done, there are new test files test_update_relation.py and test_create_relation.py

File: openslides_backend/action/action.py (hunk 4 above)
Comment (r-peschke) [reply openslides_backend/action/action.py]: outdated, this is now detected by checking the WrieRequests

File: openslides_backend/action/action_handler.py (hunk 7 above)
Comment (jsangmeister) [reply openslides_backend/action/action_handler.py:187]: Ok, my bad - this is already the upper level.

File: openslides_backend/action/mixins/create_action_with_dependencies.py (hunk 10 above)
Comment (r-peschke) [reply openslides_backend/action/mixins/create_action_with_dependencies.py:23]: done

File: openslides_backend/action/actions/motion_state/create_update_delete.py (hunk 9 above)
Comment (r-peschke) [reply openslides_backend/action/actions/motion_state/create_update_delete.py:65]: removed. Field belongs to meeting.create
The field "first_state_of_workflow_id" is since 10/2020 in the crate schema present. wiki updated [motion_state.create Payload](https://github.com/OpenSlides/OpenSlides/wiki/motion_state.create)

File: openslides_backend/action/actions/group/create.py (hunk 8 above)
Comment (r-peschke) [reply openslides_backend/action/actions/group/create.py:19]: Removed. These fields are part of meeting.create

File: openslides_backend/action/mixins/create_action_with_dependencies.py (hunk 10 above)
Comment (r-peschke) [reply openslides_backend/action/mixins/create_action_with_dependencies.py:23]: Changed. The _index _ is gone and the expected return value is now a List of dict

File: openslides_backend/models/base.py [hunk 11]
```diff
@@ -124,3 +124,17 @@ def get_properties(self, *fields: str) -> Dict[str, fields.Schema]:
         for field in fields:
             properties.update(self.get_property(field))
         return properties
+
+    def get_required_fields(self) -> Iterable[fields.Field]:
+        """
+        Yields all required fields
+        """
+        for model_field in self.get_fields():
+            if model_field.required and isinstance(model_field, fields.Field):
```
Comment (jsangmeister) [inline openslides_backend/models/base.py:133]: What is the use of the `isinstance` check here? Shouldn't this always be a `Field`?

File: tests/system/action/test_action_command_format.py [hunk 12]
```diff
@@ -81,75 +75,73 @@ def test_parse_actions_create_1_2_events(self) -> None:
...
+        self.assertEqual(str(write_requests[0].events[1]["fqid"]), "group/2")
+        self.assertEqual(str(write_requests[0].events[2]["fqid"]), "meeting/1")
+        self.assertEqual(str(write_requests[0].events[3]["fqid"]), "meeting/1")
 
     def test_create_2_actions(self) -> None:
-        self.create_model("committee/1", {"name": "test_committee"})
-        response = self.request_json(
-            [
+        self.create_model("meeting/1", {})
+        response = self.client.post(
```
Comment (jsangmeister) [inline tests/system/action/test_action_command_format.py:90]: Please use `request_json` here

File: tests/system/action/test_action_command_format.py [hunk 13]
```diff
@@ -81,75 +75,73 @@ def test_parse_actions_create_1_2_events(self) -> None:
...
                     ],
                 },
             ],
         )
         self.assert_status_code(response, 400)
         self.assertIn(
-            "Datastore service sends HTTP 400. Model 'committee/1' raises MODEL_LOCKED error.",
-            response.json["message"],
+            "Datastore service sends HTTP 400. Model \\'meeting/1\\' raises MODEL_LOCKED error.",
+            str(response.data),
```
Comment (jsangmeister) [inline tests/system/action/test_action_command_format.py:116]: Please use `request.json` here to have access to the decoded string

File: tests/system/action/test_action_command_format.py [hunk 14]
```diff
@@ -81,75 +75,73 @@ def test_parse_actions_create_1_2_events(self) -> None:
...
+        self.assert_model_not_exists("group/2")
+        self.assert_model_exists("meeting/1", {"group_ids": None})
 
     def test_create_1_2_events(self) -> None:
-        self.create_model("committee/1", {"name": "test_committee"})
-        response = self.request_multi(
-            "meeting.create",
-            [
+        self.create_model("meeting/1", {})
+        response = self.client.post(
```
Comment (jsangmeister) [inline tests/system/action/test_action_command_format.py:124]: Please use `request_multi` here

**Review by jsangmeister — CHANGES_REQUESTED** [review]
Looks good now besides a few minor things! Can be merged afterwards

File: openslides_backend/models/base.py (hunk 11 above)
Comment (r-peschke) [reply openslides_backend/models/base.py:133]: Removed! I remember there was an issue, but now all tests are successful without the isinstance.
